nature/migrations/0004_field_updates.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-10-31 09:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import nature.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nature', '0003_update_timestamp_fields'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='event',
+            name='protection_level',
+            field=models.IntegerField(choices=[(1, 'Administrators only'), (2, 'Administrators and staff'), (3, 'Public')], db_column='suojaustasoid', default=3),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='active',
+            field=models.BooleanField(db_column='voimassa', default=True),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='geometry1',
+            field=nature.models.PermissiveGeometryField(db_column='geometry1', srid=3879),
+        ),
+        migrations.RenameField(
```

Comment:
Don't know if we can do this either, the views probably would have to be redefined if we want to change the name of the column in the database.